zebra/zebra_dplane.c

```diff
@@ -885,39 +890,40 @@ void dplane_ctx_fini(struct zebra_dplane_ctx **pctx)
 	dplane_ctx_free(pctx);
 }
 
+/* Init a list of contexts */
+void dplane_ctx_q_init(struct dplane_ctx_list_head *q)
```

Comment:
Nit: this is fine, but it would be also fine to just call `dplane_ctx_list_init` instead of this function. Am I missing something?

Reply:
Because these things are actually macros, they only work if the caller can see the structs involved - and the dplane ctx is opaque to outsiders. I didn't want to relax that just to make these changes?

Reply:
Makes sense, sorry I missed that.